packages/firestore/test/integration/util/events_accumulator.ts

```diff
@@ -16,7 +16,7 @@
  */
 
 import * as firestore from '@firebase/firestore-types';
-import { Deferred } from '@firebase/util';
+import { Deferred } from '../../util/promise';
```

Comment:
Can we just use `Deferred` from `@firebase/util`?

Reply:
This is the only usage of the Deferred promise in Firestore. Since I realized this is test-only, it probably doesn't matter much. I can revert this change with my next update to this PR (I just got the CI to green after one week...)

Reply:
Removed this change.